graphql_ws/__init__.py

```diff
@@ -7,6 +7,4 @@
 __version__ = '0.3.1'
 
 
-from .base import BaseConnectionContext, BaseSubscriptionServer
-
-__all__ = ['BaseConnectionContext', 'BaseSubscriptionServer']
```

Comment:
As far as I know, this line is supposed to export only the declared functions and "avoid" any other internal import from those modules.

Reply:
There's nothing that use these imports internally, the code all references the explicit `base`. These two classes aren't really usable externally, except perhaps as the base to your own third-party server. For a project that has languished so long, I don't see that being too much of a concern though - people can just update their code to import from the explicit base module too.